queue/abstract/driver/fifottl.lua

```diff
@@ -262,18 +262,17 @@ end
 function method.take(self)
     local offset = 0
```

Comment:
Seems like `offset` is not needed anymore?

Reply:
Sure, thanks! Force-pushed.